FSJ_django20_project/FSJ/templates/FSJ/coord_student_list.html

```diff
@@ -32,5 +32,9 @@
 {% endfor %}
 </table>
```

Comment:
Adjudicator form adds user first and last name but student's don't have this information shown?

Reply:
Fixed